source/player_guide/concepts_info/concepts/army_training.rst

```diff
@@ -0,0 +1,40 @@
+.. _concept_armytraining:
+
+Army Training
+=================
```

Comment:
__suggestion__

instead of army, use the same name, that is used by the ingame button.